.php_cs.dist

```diff
@@ -1,4 +1,3 @@
-#!/usr/bin/env php
```

Comment:
This line helps IDEs with autocompletion since the `.php_cs.dist` filename doesn't have `.php` extension